Rules/UseCmdletCorrectly.cs

```diff
@@ -22,6 +23,24 @@ namespace Microsoft.Windows.PowerShell.ScriptAnalyzer.BuiltinRules
 #endif
     public class UseCmdletCorrectly : IScriptRule
     {
+        private static readonly ConcurrentDictionary<string, IReadOnlyList<string>> s_pkgMgmtMandatoryParameters =

```

Comment:
it might be useful to provide a list of what the 2 columns are. I assume the first column is a cmdlet name and the 2nd is the problematic parameters? Just a comment would do the trick.

Also, the name of the member might be more generic, like `s_BadCmdletMandatoryParameters` or some such.